NTupleAnalyzer_nanoAOD.py

```diff
@@ -10,6 +10,11 @@
 from random import randint
 from random import normalvariate
 
+# GE code is in C++. Use the root gInterpreter to import library
+gInterpreter.ProcessLine('#include "/afs/cern.ch/work/g/gmadigan/CMS/Analysis/Leptoquarks/MakeTreesStockNanoAODv6_2/LQ2Analysis13TeV/GEScaleSyst/GEScaleSyst.cc"')
```

Comment:
The fastest will be just to add them to GitHub and access them locally

Reply:
Sure. There is a command for accessing them in cmssw but I need to convert from C++ to python. There's also the GEScaleSyst repo too, so it may be better to add them to the github. I can add them for now and we can work on a more forward compatible implementation later.

Reply:
Good, but you still need to change these file paths to the new format. Make sure to test locally :)

Reply:
Of course 👍 working on it. Actually having difficulty adding the GEScaleSyst package. I might need to recopy it then commit.

Reply:
Scratch that. We only need Rochester for this. GE is only needed for the GE scale study which I have a custom analyzer for. I'll change the paths, test, and submit a new commit.